CHANGELOG.md

```diff
@@ -17,8 +17,10 @@ This website is built from the `HEAD` of the `main` branch of the theme reposito
 
 Code changes to `main` that are *not* in the latest release:
 
+- Remove the `passive` toggle by [@pdmosses] in [#1335]
```

Comment:
So sorry that I missed this in the first review, but what are your thoughts on the format + a potential clarification (not sure if the `.` helps in explaining that this is a CSS class):

```suggestion
- Remove: unused `.passive` toggle in navigation by [@pdmosses] in [#1335]
```